conference/src/main/java/org/spine/samples/lobby/conference/ConferenceServiceImpl.java

```diff
@@ -0,0 +1,206 @@
+import org.spine3.samples.lobby.conference.EditableConferenceInfo;
+import org.spine3.samples.lobby.conference.FindConferenceRequest;
+import org.spine3.samples.lobby.conference.PublishConferenceRequest;
+import org.spine3.samples.lobby.conference.PublishConferenceResponse;
+import org.spine3.samples.lobby.conference.UpdateConferenceResponse;
+import org.spine3.samples.lobby.conference.contracts.Conference;
+import org.spine3.samples.sample.lobby.conference.contracts.ConferenceCreated;
+import org.spine3.samples.sample.lobby.conference.contracts.ConferencePublished;
+import org.spine3.samples.sample.lobby.conference.contracts.ConferenceUpdated;
+import org.spine3.server.BoundedContext;
+
+import java.math.BigInteger;
+import java.security.SecureRandom;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * @author andrii.loboda
+ */
+public class ConferenceServiceImpl implements ConferenceService {
+
+
+    private final BoundedContext boundedContext;
+    private final ConferenceRepository conferenceRepository;
+
+
+    public ConferenceServiceImpl() {
+        boundedContext = getBoundedContext();
+        conferenceRepository = getRepository();
+    }
+
+
+    @Override
+    public void createConference(ConferenceInfo conferenceToCreate, StreamObserver<CreateConferenceResponse> responseObserver) {
+
+        final String accessCode = generateAccessCode();
+        final Conference conference = asConference(conferenceToCreate);
+        final Conference conferenceToPersist = conference.toBuilder()
+                                                         .setAccessCode(accessCode)
+                                                         .build();
+        conferenceRepository.store(conferenceToPersist);
+
+        sendConferenceCreatedEvent(conferenceToPersist);
+
+        final CreateConferenceResponse build = CreateConferenceResponse.newBuilder()
+                                                                       .setId(conferenceToPersist.getId())
+                                                                       .setAccessCode(accessCode)
+                                                                       .build();
+
+        responseObserver.onNext(build);
+        responseObserver.onCompleted();
+    }
+
+
+    @Override
+    public void findConference(FindConferenceRequest request, StreamObserver<Conference> responseObserver) {
+        final Conference conference = conferenceRepository.loadByEmailAndAccessCode(request.getEmailAddress(), request.getAccessCode());
+
+        responseObserver.onNext(conference);
+        responseObserver.onCompleted();
+    }
+
+    @Override
+    public void updateConference(EditableConferenceInfo request, StreamObserver<UpdateConferenceResponse> responseObserver) {
+        final Conference existingConference = conferenceRepository.load(request.getId());
+
+        checkNotNull(existingConference, "Can't finde conference with specified id: " + request.getId());
```

Comment:
o_O
Finde? Is it Danish? Then change it to:
`Kan ikke finde konference med specificeret id.`